cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,27 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+
 describe('User', () => {
-  before(() => {
+  let firstUser;
+  let secondUser;
 
-  });
+  before(() => {
+    cy.task('db:clear');
+    cy.registerNewUser().then(user => {
+       firstUser = user;
+    });
+    cy.registerNewUser().then(user => {
```

Comment:
Async operations in the before hook may not work as expected. Tests may start running before these promises resolve.